generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -645,7 +645,7 @@ _%>
               <span className="d-none d-md-inline" ><Translate contentKey="entity.action.back">Back</Translate></span>
             </Button>
             &nbsp;
-            <Button color="primary" id="save-entity" type="submit" disabled={isInvalid || updating}>
+            <Button color="primary" id="save-entity" type="submit" disabled={updating}>
```

Comment:
why is the invalid check removed? the idea was you cannot submit a form if it doesnt pass validation

Reply:
Let me double check, but I think I removed it because it was unused. Like the variable was initialized but never modified or anything.

Reply:
If you search for `isInvalid` refs in the file you only find it 2 times. The first is the declaration and the second is `disabled={isInvalid || updating}`. So yes, I think it's pretty much useless.